app/assets/javascripts/client/inventories/InventoryDataEntryCtrl.js

```diff
@@ -0,0 +1,26 @@
+(function() {
+  'use strict';
+
+  angular.module('PDRClient')
+      .controller('InventoryDataEntryCtrl', InventoryDataEntryCtrl);
+
+  InventoryDataEntryCtrl.$inject = [
+    '$scope',
+    '$modal'
+  ];
+
+  function InventoryDataEntryCtrl($scope, $modal) {
+    var vm = this;
+
+    vm.showInventoryDataEntryModal = function() {
+      vm.modalInstance = $modal.open({
+        template: '<inventory-data-entry-modal inventory="inventory" resource="resource"></inventory-data-entry-modal>',
```

Comment:
Where is it pulling the inventory and resource from?  I'm under the impression that those are strings, but I may be wrong on that.